src/main/java/Goot.java

```diff
@@ -0,0 +1,83 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+//import java.util.Arrays;
+//import java.util.ArrayList;
+public class Goot {
+    public static void main(String[] args) {
+    String DASH = "__________________________________";
+    String greet = DASH + "\nI'm Goot :3\nWhat's up?\n" + DASH;
+    System.out.println(greet);
+    File f = new File("data.txt");
+    try{
+        Task.loadFileContentsToTaskArray(f);
+    }
+    catch (IOException e){
```

Comment:
Perhaps it is better to catch the more specific exception. In this case use  FileNotFoundException instead of IOException since you only plan to catch FileNotFoundException.